README.md

```diff
@@ -364,6 +364,13 @@ _instructions for google oauth2 setup borrowed from [oauth2_proxy](https://githu
 
 For more details refer to [Yandex OAuth](https://tech.yandex.com/oauth/doc/dg/concepts/about-docpage/) and [Yandex.Passport](https://tech.yandex.com/passport/doc/dg/index-docpage/) API documentation.
 
+##### Battle.net Auth Provider
```

Comment:
Will be nice to have the actcual inststuction, like other providers have

Reply:
@umputun Added it in a new commit